packages/design-system/src/components/tooltip/tail.js

```diff
@@ -52,14 +52,14 @@ const getTailPosition = ({ placement, translateX }) => {
     case PLACEMENT.LEFT_END:
       return css`
         right: -${TAIL_WIDTH / 2 + TAIL_HEIGHT / 2 - 1}px;
-        transform: rotate(90deg);
+        transform: rotate(${isRTL ? '-90deg' : '90deg'});
```

Comment:
This was already broken in main, we just didn't notice cause we currently aren't using any right or left tooltip with a tail